wordpress/wordpress-api.ts

```diff
@@ -1,9 +1,11 @@
-
 /**
  * `V2 Endpoints /wordpress/v2`
  */
-export const WORD_PRESS_API_V2_PAGES = '/wp-json/wp/v2/pages';
+// export const WORD_PRESS_API_V2_PAGES = '/wp-json/wp/v2/pages';
+export const WORD_PRESS_API_V2_PAGES = '/wp-json/wp/v2/pages?per_page=100';
```

Comment:
It would be better if this would be:

    export const WORD_PRESS_API_V2_PAGES = (perPage: number) => `/wp-json/wp/v2/pages?per_page=${q(perPage)}`;

...where `q` is just:

```
function q (value: string) : string {
    return encodeURIComponent(value);
}
```

...and then you can define the amount of pages in the code.